db/migrate/20181119211456_add_note_to_price_policies.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddNoteToPricePolicies < ActiveRecord::Migration[5.0]
+  def change
+    change_table :price_policies do |t|
+      t.text :note
```

Comment:
@vandrijevik I went with a `text` field before our conversation today. Do you think I should switch it to `string`/`varchar`? I kind of like being more flexible in terms of giving the user enough space.

Reply:
Hmm… I’d probably put this as a `t.string :note, limit: 256`, but I don’t feel too strongly about that 256. I do think we should specify a hard limit on the column size, though!